jenkins-integration-build.sh

```diff
@@ -57,7 +57,10 @@ if [ "${USE_GO_MODULES:-}" == "false" ]; then
     go get -u -v github.com/AlekSi/gocov-xml
 else
     # Install tools to use after job has completed
-    go install -v github.com/tebeka/go2xunit@latest
+    # go2xunit will fail with 1.23 with name mismatch (try disabling parallel mode), but without any t.Parallel()
+    go install golang.org/dl/go1.22.8@latest
+    ~/go/bin/go1.22.8 download
+    ~/go/bin/go1.22.8 install -v github.com/tebeka/go2xunit@latest
```

Comment:
As discussed this isn't going to fix the issue (it seems like this issue is sporadic and not related to Go version)

```suggestion
    go install -v github.com/tebeka/go2xunit@latest
```

Reply:
I tested this on a bunch of files from Jenkins locally, and there are way more cases where this will fail with go 1.23 and not go 1.22

I do not think this is a robust solution and I think switching to go-junit-report with either a patch or post processing the xml will be the right solution. I'll file a ticket for this since we now saw this elsewhere. I do think that some of the problems came from parsing the panics in https://github.com/couchbase/sync_gateway/commit/142f2156bedfaec8e824d9724ce875f5de20818c.